llvm/include/llvm/MC/MCRegisterInfo.h

```diff
@@ -382,6 +385,10 @@ class MCRegisterInfo {
     return RegStrings + get(RegNo).Name;
   }
 
+  bool isConstant(MCRegister RegNo) const {
+    return get(RegNo).Constant && MCRegister::isPhysicalRegister(RegNo.id());
```

Comment:
Isn't an MCRegister always a physical register?

Reply:
Yes, you're right. I added this while I was testing things, my bad for forgetting to remove this check.